src/convertToObject.js

```diff
@@ -1,18 +1,23 @@
 'use strict';
 
 /**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const convertedObj = {};
+
+  sourceString
+    .split(';')
+    .map(item => item.trim().split(':'))
+    .filter(item => item.length > 1)
+    .forEach(item => {
```

Comment:
would be great to use descturcturing here to improve readability
```suggestion
    .forEach([prop, value] => {
```